platformio_orig.ini

```diff
@@ -65,6 +66,7 @@ display_library = ; set by build.py and taken from hal file
 lib_deps_lora =
     mcci-catena/MCCI LoRaWAN LMIC library @ ^3.3.0
 lib_deps_display =
+    https://github.com/lewisxhe/GxEPD.git
```

Comment:
Shouldn't we better use [GxEPD2](https://github.com/ZinggJM/GxEPD2) here?

Reply:
Ah. I am sorry. That is a remanence from my testing with another library that is specifically targeted for ttgo5. I changed it in my plattformio.ini but apparantly forgot to change it in platformio_orig.ini